src/renderer/widgets/Transfer/default/model/form-model.ts

```diff
@@ -71,6 +74,22 @@ const myselfClicked = createEvent();
 const xcmDestinationSelected = createEvent<AccountId>();
 const xcmDestinationCancelled = createEvent();
 
+const setMaxMode = createEvent<boolean>();
+const $isMaxModeEnabled = createStore(false)
+  .on(setMaxMode, (_, update) => update)
+  .reset(formInitiated);
+
+const toggleExistentialDeposit = createEvent<boolean | void>();
```

Comment:
don't use void or undefined inside eff
```suggestion
const toggleExistentialDeposit = createEvent<boolean | null>();
```